sanic/cookies.py

```diff
@@ -116,8 +116,9 @@ def encode(self, encoding):
                     ))
                 except AttributeError:
                     output.append('%s=%s' % (self._keys[key], value))
-            elif key in self._flags and self[key]:
-                output.append(self._keys[key])
+            elif key in self._flags:
+                if self[key]:
```

Comment:
This logic should actually be the same as previously. Did you try running the tests without this change, to make sure they actually fail?